runtime/gc_glue_java/configure_includes/configure_linux_riscv.mk

```diff
@@ -22,9 +22,6 @@
 
 include $(CONFIG_INCL_DIR)/configure_common.mk
 
-# The riscv64 cross toolchain is used for both 32bit and 64bit
-RISCV_CROSSTOOLS_PREFIX=riscv64-unknown-linux-gnu
```

Comment:
Rather than remove this, I suggest it should supply a default.

Reply:
The default value of  `RISCV_CROSSTOOLS_PREFIX` is set up at https://github.com/ibmruntimes/openj9-openjdk-jdk11/pull/297 as follows:
```
    ifeq (install,$(RISCV_TOOLCHAIN_TYPE))
      export RISCV_CROSSTOOLS_PREFIX := riscv64-linux-gnu
    else
      export RISCV_CROSSTOOLS_PREFIX := riscv64-unknown-linux-gnu
    endif
```

Reply:
As discussed with @keithc-ca offline, changed the script to set the default value of `RISCV_CROSSTOOLS_PREFIX` if not set up in OpenJDK.